products/green_fast_track/macros/clip_to_geom.sql

```diff
@@ -1,8 +1,11 @@
-{% macro clip_to_geom(left, right=ref("stg__nyc_boundary"), left_by="geom", right_by="geom") %}
+{% macro clip_to_geom(left, right=ref("stg__nyc_boundary"), left_by="geom", right_by="geom", crs=2263) %}
 
     SELECT
         {{ dbt_utils.star(from=left, except=[left_by]) }},
-        ST_INTERSECTION({{ left }}.{{ left_by }}, {{ right }}.{{ right_by }}) AS geom
+        ST_INTERSECTION(
+            ST_TRANSFORM({{ left }}.{{ left_by }}, {{ crs }}),
+            ST_TRANSFORM({{ right }}.{{ right_by }}, {{ crs }})
```

Comment:
Per stand up this morning, the rational to modify the macro is because it didn't work with a CTE for a stage table (usparks). And I couldn't use it directly on source because it was in a different projection. Finn and I will check in to figure out an approach.

Reply:
Would this piece of code live in the macro?

Reply:
I'm not sure how to implement this...